### ISSUE TITLE
WAVE Info tags not removed properly

### ISSUE THREAD

**Issue by sbooth** [taglib/taglib#500]
I've discovered odd behavior where tag information isn't properly removed from WAVE files.  I created an empty WAVE in audacity and verified that it had no tag information in a hex editor and using the `tagreader` example:

```
./tagreader ~/Desktop/test.wav
-- TAG (basic) --
title   - ""
artist  - ""
album   - ""
year    - "0"
comment - ""
track   - "0"
genre   - ""
-- TAG (properties) --
-- AUDIO --
bitrate     - 705
sample rate - 44100
channels    - 1
length      - 0:01
```
Then I added a dummy title and verified it was there:

`./tagwriter -t "Title" ~/Desktop/test.wav `

```
./tagreader ~/Desktop/test.wav
-- TAG (basic) --
title   - "Title"
artist  - ""
album   - ""
year    - "0"
comment - ""
track   - "0"
genre   - ""
-- TAG (properties) --
TITLE - "Title"
-- AUDIO --
bitrate     - 705
sample rate - 44100
channels    - 1
length      - 0:10
```

Interestingly, the reported length changed from 1 second (the actual length) to 10 seconds.  This is something I need to investigate further.

I then attempted to remove the title by setting it to an empty string:

`./tagwriter -t "" ~/Desktop/test.wav `

But the tag persisted:

```
./tagreader ~/Desktop/test.wav
-- TAG (basic) --
title   - "Title"
artist  - ""
album   - ""
year    - "0"
comment - ""
track   - "0"
genre   - ""
-- TAG (properties) --
TITLE - "Title"
-- AUDIO --
bitrate     - 705
sample rate - 44100
channels    - 1
length      - 0:10
```

**Comment by TsudaKageyu** [comment]
I just confirmed the issue. I will look into it on this weekend.

**Comment by TsudaKageyu** [comment]
Ok. I fixed it at #502.

**Comment by sbooth** [comment]
I confirm this fixed the problem.  Thanks!
